packages/plugins/src/plugins/metronome/metronomeJob.ts

```diff
@@ -0,0 +1,159 @@
+import { EvmNetworkIdType } from '@sonarwatch/portfolio-core';
+import { Address, ContractFunctionConfig, PublicClient } from 'viem';
+import { Cache } from '../../Cache';
+import { Job, JobExecutor } from '../../Job';
+import { getEvmClient } from '../../utils/clients';
+import { lendingAbi } from './abi';
+import {
+  PLATFORM_ID,
+  METRONOME_MARKETS_KEY,
+  MARKET_ADDRESS,
+} from './constants';
+import { MetronomeMarket, UnderlyingInfo } from './types';
+import { mapToSuccess } from '../../utils/octav/mapToSuccess';
+import {
+  executeAndSplitMulticall,
+  executeGroupedMulticall,
+} from '../../utils/octav/splitMulticallResult';
+
+export default function getMetronomeJob(networkId: EvmNetworkIdType): Job {
+  const client: PublicClient = getEvmClient(networkId);
+
+  const executor: JobExecutor = async (cache: Cache) => {
+    // Fetch Deposit and Debt Token Lists
+    const marketCalls = [
+      {
+        address: MARKET_ADDRESS,
+        abi: lendingAbi,
+        functionName: 'getDepositTokens',
+      },
+      {
+        address: MARKET_ADDRESS,
+        abi: lendingAbi,
+        functionName: 'getDebtTokens',
+      },
+    ];
+
+    const [depositTokensResult, debtTokensResult] =
+      await executeAndSplitMulticall(client, marketCalls);
+
+    const depositTokens = mapToSuccess(depositTokensResult).flat() as Address[];
+    const debtTokens = mapToSuccess(debtTokensResult).flat() as Address[];
+
+    // Get underlying deposit tokens and synthetic debt tokens
+    const allCalls = [
+      ...depositTokens.map((token) => ({
+        address: token,
+        abi: lendingAbi,
+        functionName: 'underlying' as const,
+      })),
```

Comment:
```suggestion
        functionName: 'underlying',
      } as const)),
```

Wasn't the `as const` misplaced here ?